### PR TITLE
Add activation and activation_info modules

### PR DESCRIPTION
**Pull request by alinabuzachis**
Add activation and activation_info modules

### CODE REVIEW

**Review by ssbarnea — APPROVED** [review]
This LGTM but keep in mind that I do not have much experience with EDA itself.

File: tests/integration/targets/activation/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        enabled: False
+        awx_token_name: "{{ awx_token_name }}"
+      register: _result
+
+    - name: Check rulebook activation creation
+      assert:
+        that:
+          - _result.changed
+
+    - name: Create a new rulebook activation again
```
Comment (matoval) [inline tests/integration/targets/activation/tasks/main.yml:170]: I getting an error Create a new rulebook activation again
`fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to create/update rulebook activation: HTTP error {'detail': 'Method \"PATCH\" not allowed.'}"}`

Comment (alinabuzachis) [reply tests/integration/targets/activation/tasks/main.yml:168]: I added a note about the fact that the module does not support PATCH and also handled that at the module level. Let me know if that makes sense to you. Thanks.

Comment (matoval) [reply tests/integration/targets/activation/tasks/main.yml:168]: Makes sense, but I'm getting an error about the assert checking result message:

`[WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}.
Found: 'A rulebook activation with name: {{ activation_name }} already exists. The module does not support
modifying a rulebook activation.' in _result.msg
fatal: [localhost]: FAILED! => {"msg": "The conditional check ''A rulebook activation with name: {{ activation_name }} already exists. The module does not support modifying a rulebook activation.' in _result.msg' failed. The error was: Conditional is marked as unsafe, and cannot be evaluated."}`

Comment (alinabuzachis) [reply tests/integration/targets/activation/tasks/main.yml:168]: Removed jinja templating from the assert.

**Review by matoval — APPROVED** [review]
Ran activation integration tests and all ran successfully

**Pull request merged**